README.md

```diff
@@ -78,6 +78,7 @@
 * [KuberNix](https://github.com/saschagrunert/kubernix) - Single-dependency Kubernetes clusters via Nix packages.
 * [morph](https://github.com/DBCDK/morph) - A tool for managing existing NixOS hosts.
 * [Nixery](https://github.com/tazjin/nixery) - A Docker-compatible container registry which builds images ad-hoc via Nix.
+* [Nixinate](https://github.com/MatthewCroughan/nixinate) - A flake based deployment tool.
```

Comment:
The tool itself looks good. Maybe adding a bit more to this entry's description might be good though.

```suggestion
* [Nixinate](https://github.com/MatthewCroughan/nixinate) - A Nix flake library to provide app outputs for managing existing NixOS hosts over SSH.
```

Reply:
well that's a matter of debate.

Reply:
It creates an executable/runnable (shell script) that is able to be used via `nix run`, but I think it's more suitable to call it a library, since I will be providing more functions can generate those outputs. I'd call it a Nix deployment library, once that's done, but right now it's very much a tool since it can only do one single thing, which is generate that single script.

Reply:
ok, thats a good appraisal , looks goot LGTM. @houstdav000  shall we get this merged.